lib/db/embeddings_db.dart

```diff
@@ -0,0 +1,65 @@
+import "dart:io";
+
+import "package:isar/isar.dart";
+import 'package:path_provider/path_provider.dart';
+import "package:photos/core/event_bus.dart";
+import "package:photos/events/embedding_updated_event.dart";
+import "package:photos/models/embedding.dart";
+
+class EmbeddingsDB {
+  late final Isar _isar;
+
+  EmbeddingsDB._privateConstructor();
+
+  static final EmbeddingsDB instance = EmbeddingsDB._privateConstructor();
+
+  Future<void> init() async {
+    final dir = await getApplicationDocumentsDirectory();
+    _isar = await Isar.open(
+      [EmbeddingSchema],
+      directory: dir.path,
+    );
+    // TODO: _clearDeprecatedStore(dir);
```

Comment:
Remove?

Reply:
Was hedging, uncommented, will drop the ObjectBox DB.